### DISCUSSION TITLE
Import keywords?

### DISCUSSION THREAD

**Discussion by tonblom** [mealie-recipes/mealie#1030] [Ideas]
I searched but could not find it. Is it possible to also import keywords from recipes. If not it would be helpful to select keywords (not all) and if they don't exist to also add a new keywords to mealie

**Comment by hay-kot** [comment]
This is not currently implemented, I found that some recipes would have 10+ keywords and I like to be very careful of what I use for categories/tags. I could see a checkable box on the importer to import keywords as tags though. I'm open to the idea.

**Comment by hay-kot** [comment]
I've added a task to track progress on this

- https://github.com/hay-kot/mealie/issues/1105
